config/main.py

```diff
@@ -6388,7 +6388,8 @@ def remove_reasons(counter_name, reasons, verbose):
 @click.option('-ydrop', metavar='<yellow drop probability>', type=click.IntRange(0, 100), help="Set yellow drop probability")
 @click.option('-gdrop', metavar='<green drop probability>', type=click.IntRange(0, 100), help="Set green drop probability")
 @click.option('-v', '--verbose', is_flag=True, help="Enable verbose output")
-def ecn(profile, rmax, rmin, ymax, ymin, gmax, gmin, rdrop, ydrop, gdrop, verbose):
+@multi_asic_util.multi_asic_click_option_namespace
+def ecn(profile, rmax, rmin, ymax, ymin, gmax, gmin, rdrop, ydrop, gdrop, verbose, namespace):
```

Comment:
I'm a little confused, without adding option `-n` here, how do we use this namespace? or is it added in an above layer

Reply:
The `-n` option is added by the multi_asic_click_option_namespace. For reference here is how it does it: https://github.com/sonic-net/sonic-utilities/blob/317e649514c9b205849ffb5ea96a6a233e38290c/utilities_common/multi_asic.py#L106

Reply:
@wenyiz2021 are you asking about the decorator `@multi_asic_util.multi_asic_click_option_namespace`? Or are you asking about passing `-n` to `scripts/ecnconfig`?  

We want to add the `namepace` parameter to both the`config ecn` and `show ecn` CLI commands to teach them to operate on multi-asic platforms.

Reply:
got it now.